gssapi/_win_config.py

```diff
@@ -17,12 +18,16 @@
 )
 #: Download location for KfW
 KFW_DL = "https://web.mit.edu/KERBEROS/dist"
+is64bits = sys.maxsize > 2**32
```

Comment:
This is only referenced once, so it's not really clearer to have it be its own variable.

Reply:
I do agree about using a variable instead of the logic in general.

But - using the statement `sys.maxsize > 2**32` will puzzle people in long term, what's going on here.
And - that's the reason - I have used a variable here.

 I update the pull request to remove the variable use here.